doltpy/etl/dolt_loader.py

```diff
@@ -1,46 +1,36 @@
 import argparse
 from doltpy.core import Dolt
 from typing import List
-from doltpy.etl.loaders import load_to_dolt, resolve_function, DoltTableLoader, resolve_branch
+from doltpy.etl.loaders import resolve_function, DoltLoaderBuilder
 import logging
 from doltpy.etl.cli_logging_config_helper import config_cli_logger
 
 logger = logging.getLogger(__name__)
 
 
-def loader(loaders: List[DoltTableLoader], dolt_dir: str, commit: bool, message: str, dry_run: bool, branch: str):
+def loader(loader_builder: DoltLoaderBuilder, dolt_dir: str, dry_run: bool):
     logger.info(
         '''Commencing load to Dolt with the following options, and the following options
```

Comment:
Should just be "Commencing load to Dolt with the following options"?

Reply:
Oh dear...